.github/workflows/workflow.yml

```diff
@@ -14,18 +14,33 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         node-version: ['10.18.0', '*']
-        exclude:
-          - os: macOS-latest
+        # The buildbot pins Netlify Build's dependencies, like `npm ci`.
+        # But other consumers do not, like `npm install`.
+        # So we test both.
+        install-command: ['npm ci', 'npm install']
+        include:
+          - os: ubuntu-latest
+            node-version: '*'
+            install-command: npm install
+          - os: ubuntu-latest
+            node-version: '*'
+            install-command: npm ci
+          - os: ubuntu-latest
             node-version: '10.18.0'
+            install-command: npm ci
+          - os: macOS-latest
+            node-version: '*'
+            install-command: npm ci
           - os: windows-latest
-            node-version: '10.18.0'
+            node-version: '*'
+            install-command: npm ci
       fail-fast: false
     steps:
       # Sets an output parameter if this is a release PR
       - name: Check for release
         id: release-check
         run: echo "::set-output name=IS_RELEASE::true"
-        if: "${{ startsWith(github.head_ref, 'release-') }}"
+        if: ${{ startsWith(github.head_ref, 'release-') }}
```

Comment:
`${{ }}` does not need quote in YAML. Some lines in this file use no quotes, others `"`, others `'`. This removes quotes for consistency purpose.